channel_http.py

```diff
@@ -117,6 +117,15 @@ def render_POST(self, request):
                 additional_info = {'Slack Log Data': {k:v for k,v in request.args.iteritems() if k not in ['user_agent', 'ip']}}
                 self.dispatch(canarydrop=canarydrop, src_ip=src_ip, useragent=useragent, additional_info=additional_info)
                 return self.GIF
+
+            if canarydrop._drop['type'] == 'aws_keys':
+                canarydrop._drop['hit_time'] = datetime.datetime.utcnow().strftime("%s.%f")
+                useragent = request.args.get('user_agent', [None])[0]
```

Comment:
so for this line 
`useragent = request.args.get('user_agent', [None])[0]`. I get that the default is `[None]` and so we would want the "first" element of that; but if we do get `user_agent`, does that not return a string? In which case we would get the first letter?

Reply:
Also isnt this done in the POST? and in the aws lambda code we doing a GET?